### ISSUE TITLE
Support for Angular 4

### ISSUE THREAD

**Issue by robinmaple** [WinkelLabs/angular-fan-menu#2]
Is this compatible with Angular 4? If not, any recent plan to support that? Thanks!

**Comment by ghost** [comment]
@CuppaLabs Yes! Can you please develop the same version for Angular2 or Angular4?
